src/InterfaceLimits.jl

```diff
@@ -114,51 +167,65 @@ end
 function add_injector_constraint!( # for buses that have loads and generators
     m::Model,
     injector_type::Type{StaticInjection},
-    p_var; #injection variable,
+    p_var, #injection variable,
+    hvdc_inj;
     ldf_lim = nothing,
     max_gen = nothing, # pass this if enforce_gen_limits = true
 )
     if !isnothing(ldf_lim)
-        @constraint(m, p_var >= ldf_lim)
+        @constraint(m, p_var - hvdc_inj >= ldf_lim)
         !isnothing(max_gen) && (max_gen += ldf_lim)
     end
-    !isnothing(max_gen) && @constraint(m, p_var <= max_gen)
+    !isnothing(max_gen) && @constraint(m, p_var - hvdc_inj <= max_gen)
 end
 
 function add_injector_constraint!( # for buses that have loads only
     m::Model,
     injector_type::Type{ElectricLoad},
-    p_var; #injection variable,
+    p_var, #injection variable,
+    hvdc_inj;
     ldf_lim = nothing,
     max_gen = nothing, # pass this if enforce_gen_limits = true
 )
     if !isnothing(ldf_lim)
-        @constraint(m, p_var == ldf_lim)
+        @constraint(m, p_var - hvdc_inj == ldf_lim)
     else
-        @constraint(m, p_var <= 0.0)
+        @constraint(m, p_var - hvdc_inj <= 0.0)
     end
+
 end
 
 function add_injector_constraint!( # for buses that have generators only
     m::Model,
     injector_type::Type{Generator},
-    p_var; #injection variable,
+    p_var, #injection variable,
+    hvdc_inj;
+    ldf_lim = nothing,
+    max_gen = nothing, # pass this if enforce_gen_limits = true
+)
+    @constraint(m, p_var - hvdc_inj >= 0.0)
+    !isnothing(max_gen) && @constraint(m, p_var - hvdc_inj <= max_gen)
+end
+
+function add_injector_constraint!( # for buses that have converters only
+    m::Model,
+    injector_type::Type{InterconnectingConverter},
+    p_var, #injection variable,
+    hvdc_inj;
     ldf_lim = nothing,
     max_gen = nothing, # pass this if enforce_gen_limits = true
 )
-    @constraint(m, p_var >= 0.0)
-    !isnothing(max_gen) && @constraint(m, p_var <= max_gen)
+    @constraint(m, p_var - hvdc_inj == 0.0)
```

Comment:
In line 233 in function add_variables! we define @variable(m, P[inames, get_name.(union(gen_buses, load_buses))])

This means p_var only exists for gen buses and load buses so I think will give an error trying to index P[iname, hvdc_bus]. Should we change the definition of P in line 233 to:
@variable(m, P[inames, get_name.(union(gen_buses, load_buses, **hvdc_buses**))])